Chrome Extension manifest.json in .gitignore

**Describe the bug**

I'm trying to setup `auto` to publish a Chrome Extension. I've added a Github Action that builds the extension in a `./build` folder before running `npx auto shipit`. This is the folder that will be compressed in a ZIP to be published and it's added to `.gitignore` since I don't want to commit it.

The problem is that when `npm auto shipit` runs seems it's trying to add the manifest from this folder but fails because the folder is in `.gitignore`.

Any idea how to fix this and still have the `build` folder in `.gitignore`?

**Error Message**
```
Error: Running command 'git' with args [add, ./build/manifest.json] failed

The following paths are ignored by one of your .gitignore files:
build/manifest.json
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"

    at ChildProcess.<anonymous> (/home/runner/work/client-extension/client-extension/node_modules/@auto-it/core/dist/utils/exec-promise.js:49:24)
    at ChildProcess.emit (events.js:310:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
##[error]Process completed with exit code 1.

```
**.autorc**

```json
{
  "plugins": [
    "released",
    [
      "chrome",
      {
        "id": "xxxxxxxxxxxxxxxxxxxxxxxxxx",
        "build": "./build",
        "manifest": "./build/manifest.json"
      }
    ]
  ]
}
```

**Github Action**
```yaml
name: Release

on:
  # Trigger the workflow on push or pull request,
  # but only for the master branch
  push:
    branches:
      - master

jobs:
  release:
    runs-on: ubuntu-latest
    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
    steps:
      - uses: actions/checkout@v2
        with:
          token: ${{ secrets.TOKEN_GITHUB }}
          submodules: "recursive"

      - name: Prepare repository
        run: git fetch --unshallow --tags

      - name: Use Node.js 12.x
        uses: actions/setup-node@v1
        with:
          node-version: 12.x

      - name: Cache node modules
        uses: actions/cache@v1
        env:
          cache-name: cache-node-modules
        with:
          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
          restore-keys: |
            ${{ runner.os }}-build-${{ env.cache-name }}-
            ${{ runner.os }}-build-
            ${{ runner.os }}-

      - name: Create Release
        env:
          GITHUB_TOKEN: ${{ secrets.TOKEN_GITHUB }}
          CLIENT_ID: ${{ secrets.DEV_CLIENT_ID }}
          CLIENT_SECRET: ${{ secrets.DEV_SECRET }}
          REFRESH_TOKEN: ${{ secrets.DEV_REFRESH_TOKEN }}
        run: |
          npm install
          npm run build
          npx auto shipit

```

--- a/plugins/chrome/src/index.ts
+++ b/plugins/chrome/src/index.ts
@@ -5,7 +5,7 @@ import {
   validatePluginConfiguration,
 } from "@auto-it/core";
 import * as fs from "fs";
-import { inc, ReleaseType } from "semver";
+import { inc, ReleaseType, gte } from "semver";
 import { promisify } from "util";
 import * as t from "io-ts";
 
@@ -52,6 +52,21 @@ export default class ChromeWebStorePlugin implements IPlugin {
 
   /** Tap into auto plugin points. */
   apply(auto: Auto) {
+    /** Get the latest tag in the repo, if none then the first commit */
+    const getVersion = async () => {
+      try {
+        const manifest = await getManifest(this.options.manifest);
+        const latest = await auto.git!.getLatestTagInBranch();
+        const version = gte(manifest.version, latest)
+          ? manifest.version
+          : latest;
+
+        return auto.prefixRelease(version);
+      } catch (error) {
+        return auto.prefixRelease("0.0.0");
+      }
+    };
+
     auto.hooks.validateConfig.tapPromise(this.name, async (name, options) => {
       if (name === this.name || name === `@auto-it/${this.name}`) {
         return validatePluginConfiguration(this.name, pluginOptions, options);
@@ -126,14 +141,8 @@ export default class ChromeWebStorePlugin implements IPlugin {
     });
 
     auto.hooks.getPreviousVersion.tapPromise(this.name, async () => {
-      const manifest = await getManifest(this.options.manifest);
-      const version = auto.prefixRelease(manifest.version);
-
-      auto.logger.verbose.info(
-        `${this.name}: Got previous version from manifest.json`,
-        version
-      );
-
+      const version = await getVersion();
+      auto.logger.verbose.info(`${this.name}: Got previous version:`, version);
       return version;
     });
 
@@ -143,10 +152,12 @@ export default class ChromeWebStorePlugin implements IPlugin {
       );
     });
 
-    auto.hooks.version.tapPromise(this.name, async (version) => {
+    auto.hooks.version.tapPromise(this.name, async (bump) => {
+      const version = await getVersion();
+
       // increment version
       const manifest = await getManifest(this.options.manifest);
-      manifest.version = inc(manifest.version, version as ReleaseType);
+      manifest.version = inc(version, bump as ReleaseType);
       await writeFile(
         this.options.manifest,
         JSON.stringify(manifest, undefined, 2)

--- a/plugins/chrome/src/index.ts
+++ b/plugins/chrome/src/index.ts
@@ -9,6 +9,7 @@ import { inc, ReleaseType, gte } from "semver";
 import { promisify } from "util";
 import * as t from "io-ts";
 
+const copyFile = promisify(fs.copyFile);
 const readFile = promisify(fs.readFile);
 const writeFile = promisify(fs.writeFile);
 
@@ -162,6 +163,8 @@ export default class ChromeWebStorePlugin implements IPlugin {
         this.options.manifest,
         JSON.stringify(manifest, undefined, 2)
       );
+
+      await copyFile(this.options.manifest, this.options.build);
 
       // commit new version
       await execPromise("git", ["add", this.options.manifest]);

